MSAL/src/MSALPublicClientApplication.m

```diff
@@ -114,11 +107,46 @@ + (void)load
     MSIDNotifications.webAuthDidReceiveResponseFromBrokerNotificationName = MSALWebAuthDidReceieveResponseFromBroker;
 }
 
+#pragma mark - Properties
+- (BOOL)validateAuthority { return YES; }
+- (void)setValidateAuthority:(BOOL)validateAuthority
+{
+    if ([self.configuration.authority isKindOfClass:MSALADFSAuthority.class])
```

Comment:
Additionally, validate authority is a global flag and shouldn't be applied only to default authority in the PCA, since it can be used with multiple authorities.

Reply:
sure, and changed.